src/vscode/notebook/DatabricksKernel.ts

```diff
@@ -184,6 +180,9 @@ export class DatabricksKernel implements vscode.NotebookController {
 				execution.end(false, Date.now());
 				return;
 		}
+
+		console.log("run", commandText, language)
```

Comment:
I removed the log. I didn't intend to commit it.

I tested the notebook on Databricks and it will indeed misinterpret the cell as Python and requires a `%sql` magic. This looks like a bug on Databricks side to though.

Reply:
I do not consider this a bug in Databricks - if you have a python notebook and try to run a SQL statement without the `%sql` magic, you will get an error. The magic basically defines which language to use
in this PR you added a "smarter" way of deriving the language to use in VSCOde by using the `document.languageId` but this information is not available in Databricks and I also do not think this will change in the near future 

this raises the question whether this PR makes sense at all considering it allows you create a notebook that runs in VSCode but not anymore on Databricks - what do you think?

Reply:
@gbrueckl I agree. If it is required by Databricks it should be required by this extension. The magic is absolutely a feature, not a bug, made by Databricks. Considering this change will allow for notebooks that seem correct, but don't work when ran on Databricks directly, I believe this PR should be closed